microsoft/surface/default.nix

```diff
@@ -1,8 +1,6 @@
-{ config, lib, pkgs, ... }:
-{
-  imports = [
-    ./kernel
-    ./firmware
-    ./hardware_configuration.nix
-  ];
+{ config, lib, pkgs, ... }: {
+  imports = [ ./kernel ./firmware ./hardware_configuration.nix ];
+
+  environment.systemPackages = with pkgs; [ surface-control ];
```

Comment:
I guess we need to wait until the channel moved.

Reply:
If you mean a stable release: no need, this configuration already requires the use of unstable.
If you mean unstable: yeah.

Reply:
No. I mean unstable.

Reply:
How long does that usually take?

Reply:
It should be already there. Let's restart the test.